packages/extension-chakra-storefront/src/components/image-gallery/index.jsx

```diff
@@ -35,17 +35,18 @@ const SMALL = 'small'
  * you are waiting for product data to be returnd from the server.
  */
 export const Skeleton = ({size}) => {
-    const styles = useMultiStyleConfig('ImageGallery', {size})
+    const recipe = useSlotRecipe({key: 'imageGallery'})
+    const styles = recipe({size})
 
     return (
         <Box data-testid="sf-image-gallery-skeleton">
```

Comment:
We are defining the `container` slot in theme but we are not using it tin the component.

https://github.com/SalesforceCommerceCloud/pwa-kit/blob/c8a13ae49543b8d2fb0dc2eb96fe94d7ac4e69b1/packages/extension-chakra-storefront/src/theme/components/project/image-gallery.js#L11

Are we missing using it here:
```suggestion
        <Box data-testid="sf-image-gallery-skeleton" css={styles.container}>
```